src/main/java/org/broadinstitute/hellbender/engine/GATKTool.java

```diff
@@ -660,19 +668,34 @@ private void checkToolRequirements() {
      * @return SAMFileWriter
      */
     public final SAMFileGATKReadWriter createSAMWriter(final File outputFile, final boolean preSorted) {
-        if (!hasReference() && IOUtils.isCramFile(outputFile)) {
+        return createSAMWriter(
+            (null == outputFile ? null : outputFile.toPath()),
```

Comment:
Per Louis' old suggestion below, you can just wrap the File arg as "Utils.nonNull(file).toPath()" so we don't get NPEs downstream.

Reply:
done.